frontend/src/Charts/Chart.js

```diff
@@ -44,6 +49,75 @@ const dataTypes = [
   },
 ];
 
+const rangeArray = [
+  {
+    label: 'Mai nap',
+    hasCustomRendering: false,
+    range: () => ({
+      startDate: new Date('2021-04-30'),
```

Comment:
Ez csak egy megjegyzes, ha majd lenne idotok:
Meg fix adatokkal is szebb lenne, ha lenne egy fix date-es az osszes tobbi paramot ahhoz viszonyitva kalkulalnatok ki.